spec/requests/users_spec.rb

```diff
@@ -12,9 +12,8 @@
 
   describe 'GET /users/:user_id' do
     it 'renders the users by id' do
-      get '/users/74'
-      expect(response).to have_http_status(:ok)
-      expect(response).to render_template(:show)
+      user_id = 7
```

Comment:
- [ ] Great work so far 👍 but i guess while making the changes you have removed two matchers **have_http_status** and **render_template** Kindly put them back to ensure that your app is tested according to the requirements 😊